src/website/index.js

```diff
@@ -31,9 +32,11 @@ simulations(true);
 
 render(
   <BrowserRouter>
-    <ThemeProvider>
-      <App demos={demos} />
-    </ThemeProvider>
+    <ScrollToTop>
```

Comment:
Can this go inside of ThemeProvider rather than around it or perhaps just be included inside the App rather than as a separate component?

Reply:
Good call. I changed App to a class, added the relevant code to its `componentDidUpdate`, and exported `withRouter(App)`.